### ISSUE TITLE
Expansion of user deliverable

### ISSUE THREAD

**Issue by aDotNetDeveloper** [aaronpowell/Chauffeur#43]
Hi Aaron,

I have a branch ready for a pull request on the Chauffeur repository
[https://github.com/aDotNetDeveloper/Chauffeur](url)

This branch has has the following enhancements to the user deliverable
```
change-name <username> <new username>
  Changes the user name for a given user.
change-loginname <username> <new loginname>
  Changes the login name for a given user.```

regards
Wayne

**Comment by aaronpowell** [comment]
Send the PR over :)

**Comment by aDotNetDeveloper** [comment]
[https://github.com/aDotNetDeveloper/Chauffeur/pull/1](url)

**Comment by aaronpowell** [comment]
Just had a quick look, it would seem that you've created a PR between your branch and the master branch within your own fork, not against my repository.

You'll need to open the pull request against this repo for me to be able to review/merge the changes.

**Comment by aDotNetDeveloper** [comment]
yep tried both, couldnt add a branch on your repo and couldnt reference your master in PR.....

**Comment by aaronpowell** [comment]
You'll probably need to close the PR on your own fork to make it simpler, but essentially it's following the process here: https://help.github.com/articles/creating-a-pull-request/

From your fork you can open a PR to my repo against the `master` branch.

**Comment by aDotNetDeveloper** [comment]
PR from fork is there.


W

**Comment by aaronpowell** [comment]
Closed with PR #44
